freedomofpress/securedrop-export
# Provides usb preflight checks and standardizes error messages

Closes https://github.com/freedomofpress/securedrop-workstation/issues/281
Closes https://github.com/freedomofpress/securedrop-workstation/issues/264


### Test plan
Testing the changes to this branch requires a specific branch of securedrop-workstation.

- check out https://github.com/freedomofpress/securedrop-workstation/pull/305
- Build package from this branch and install in sd-export-template
- Copy securedrop-export-<version>.deb to sd-export-template and install the package
- create metadata.json and sd-export archive based on the new functionality: `{"device":"usb-test"}` and `{"device":"disk-test"}`
- [ ] pre-flight test works as expected based on the usb device/drive connected to the export dedicated port
